doc/ruleSets/PSR12.rst

```diff
@@ -17,6 +17,8 @@ Rules
 - `lowercase_cast <./../rules/cast_notation/lowercase_cast.rst>`_
 - `lowercase_static_reference <./../rules/casing/lowercase_static_reference.rst>`_
 - `new_with_braces <./../rules/operator/new_with_braces.rst>`_
+  config:
+  ``['anonymous_class' => false]``
```

Comment:
:-1: to change the set configuration due to argument brought by `@kubawerlos`

Reply:
Reverted.